lib/config/db/couchdb-get-vendor.js

```diff
@@ -1,30 +1,37 @@
-module.exports = checkVendor
+module.exports = getVendor
 
-var findKey = require('lodash').findKey
+var defaults = require('lodash').defaults
 
 var toCouchDbUrl = require('../../utils/pouchdb-options-to-couchdb-url')
 var removeAuth = require('../../utils/remove-auth-from-url')
 
-function checkVendor (state, couch, callback) {
+function getVendor (state, couch, callback) {
   couch({url: '/'}, function (error, response, data) {
     if (error || (response && response.statusCode !== 200)) {
       var url = toCouchDbUrl(state.db.options)
       return callback(new Error('Could not find CouchDB at ' + removeAuth(url)))
     }
 
-    var vendor = findKey(data, function (property) {
-      return /^welcome/i.test(property)
+    var vendor = defaults(data.vendor, {
+      type: getVendorType(data.vendor)
     })
 
-    if (vendor !== 'couchdb') {
+    if (vendor.type  !== 'couchdb') {
       state.server.log(
         ['database', 'warn'],
         'You are not running an official CouchDB distribution, ' +
-        'but "' + vendor + '". ' +
+        'but "' + vendor.name + '". ' +
         'This might not be fully supported. Proceed at your own risk.'
       )
     }
 
-    callback(null)
+    callback(vendor)
   })
 }
+
+
+function getVendorType (vendor) {
+  if(vendor.name === 'The Apache Software Foundation') return 'couchdb'
```

Comment:
But I do agree about the `GET /_config` check... We probably should rely on the vendor name to know if we can read/write `/_config`.

The current solution does "enable" cloudant, however any changes to their vendor name in the future would mean updating `@hoodie/server`, and it may be possible for cloudant to enable `/_config` endpoint in the future. I wonder if we deal with these situations when they arise?